### DISCUSSION TITLE
Welcome to PegaDevTools Discussions!

### DISCUSSION THREAD

**Discussion by marcin-l** [marcin-l/PegaDevTools#16] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Honey0987654321** [comment]


**Reply by marcin-l** [reply]
@Honey0987654321 great that you do! Head over to Pega Academy and check out the [System Architect certification path ](https://academy.pega.com/topic/pega-platform-and-robotics-learning-paths/v1#p-03ba3f93-f912-41c6-bfc5-b3703c0e9030).
